cylc/flow/cfgspec/globalcfg.py

```diff
@@ -1230,9 +1233,7 @@ def default_for(
 
                 .. versionadded:: 8.0.0
 
-                   {PLATFORM_REPLACES.format(
-                       "[job]batch submit command template"
-                    )}
+                   {replaces}
```

Comment:
Why is this being changed?

Reply:
Just for tidiness I imagine. Otherwise it could just have fixed the over-indented hanging line `)}` by one space

Reply:
Tidier to avoid an intermediate var?

Reply:
Tidier to raise the line limit to >=120 IMHO 😄

Reply:
I was struggling to find the indentation that the linter was expecting.